go/vt/mysqlctl/backupstorage/interface.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package backupstorage contains the interface and file system implementation
+// of the backup system.
+package backupstorage
+
+import (
+	"flag"
+	"io"
+
+	log "github.com/golang/glog"
+)
+
+var (
+	// BackupStorageImplementation is the implementation to use
+	// for BackupStorage. Exported for test purposes.
+	BackupStorageImplementation = flag.String("backup_storage_implementation", "", "which implementation to use for the backup storage feature")
+)
+
+// BackupHandle describes an individual backup.
+type BackupHandle interface {
+	// Bucket is the location of the backup. Will contain keyspace/shard.
+	Bucket() string
+
+	// Name is the individual name of the backup. Will contain
+	// tabletAlias-timestamp.
```

Comment:
Fixed in last commit, now backup name looks like:
test_nj-0000062345.2015-05-21.163734